gluten-core/src/main/scala/org/apache/spark/rpc/GlutenDriverEndpoint.scala

```diff
@@ -40,6 +40,8 @@ class GlutenDriverEndpoint extends IsolatedRpcEndpoint with Logging {
   private val driverEndpoint: RpcEndpointRef =
     rpcEnv.setupEndpoint(GlutenRpcConstants.GLUTEN_DRIVER_ENDPOINT_NAME, this)
 
+  // TODO(yuan): get thread cnt from spark context
+  override def threadCount(): Int = 1
```

Comment:
@zhouyuan Why need this change both in GlutenDriverEndpoint and GluenExecutroEndpoint?

Reply:
`IsolatedRpcEndpoint ` introduced a virtual threadCount func